core/raydp-main/pom.xml

```diff
@@ -61,6 +61,10 @@
           <groupId>org.apache.ivy</groupId>
           <artifactId>ivy</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
Spark is a provided dependency here, we shouldn't need to exclude as the class file won't be packaged

Reply:
Thanks for your review! Adding the exclusion here is to resolve security issues scanned by Synk before 1.6 release.

Just as you said, it won't affect users no matter which Spark version was used.

Reply:
ah gotcha!